announcementsAPI.json

```diff
@@ -288,6 +288,58 @@
         }
       }
     },
+    "/announcements/like": {
+      "put": {
+        "tags": [
+          "AnnouncementsEndpoints"
+        ],
+        "summary": "Add announcement to liked announcements for adopter provided in token. Requires adopter role.",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/PutAdopterLikeAnnouncementRequest"
+              }
+            }
+          },
+          "required": true
+        },
+        "responses": {
```

Comment:
What if the announcement with the given id isn't found? I think we should return 404/400 then.

Reply:
My thoughts about this endpoint were that we don't care if announcement exists or doesn't exists and when adopter is authorized we always return OK status, because we won't show other messages to the user. But I can add this status to the responses if you think it's necessary

Reply:
Okey, on second thought I will add this to make sure that API is complete